cmd/placement-translator/main.go

```diff
@@ -156,19 +153,12 @@ func main() {
 		os.Exit(25)
 	}
 
-	// Get client config for view of APIExport of edge API
-	edgeViewConfig, err := configForViewOfExport(ctx, espwRestConfig, "edge.kubestellar.io")
+	edgeClientset, err := ksclientset.NewForConfig(espwRestConfig)
 	if err != nil {
-		logger.Error(err, "Failed to create client config for view of edge APIExport")
+		logger.Error(err, "failed to create provider clientset from config")
```

Comment:
Why was "Failed" changed to "failed"? Log messages are normally capitalized like an English sentence.

Reply:
Changed it back. Error strings should not be capitalized in go, not the logging.